UpdateVersion.py

```diff
@@ -0,0 +1,5 @@
+with open("VERSION", mode="r+") as f:
+  version = f.read().split(".")
+  version[-1] = str(int(version[-1]) + 1)
+  f.seek(0)
+  f.write(".".join(version))
```

Comment:
do we want to have a standard of spaces vs. tabs, and if so, # of spaces? I prefer four

also might not cause issues for us, but it's [considered good style](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to have a newline at the end of every file

Reply:
yeah, you're right. I prefer 2, but pep says 4. argh.